src/main/java/org/jboss/remoting3/RemotingXmlParser.java

```diff
@@ -105,11 +114,20 @@ private static void parseEndpointElement(final ConfigurationXMLStreamReader read
                                 parseConnectionsElement(reader, builder);
```

Comment:
@tadamski options (of type org.xnio.Options or org.jboss.remoting3.RemotingOptions) can be configured at the endpoint level to act as default connection options, or at the connection level, where they represent URI-specific connection options. Don't we need to make the same adjustment to the connection element in wildfly-config.xml as you have done for the endpoint element?

Reply:
@rachmatowicz +1
Plus, we need the xsd file for 5.2 namespace

Reply:
@fl4via @rachmatowicz 
I have added the xsd file. Hmm, my understanding is that currently default option map configured at endpoint is used both on Endpoint and Connection. I can implement that, but I'm wondering whether it is worth additional complexity. Btw, if implemented, do you think that those two maps should be somewhat related? (f.e. default connection map inherits options for default endpoint map)

Reply:
@tadamski we don't need to have a default connection map, but at least make sure that we have the option element in wildfly-config, plus allow the configuration of the max inbound and outbound channels there as well